src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
+@import "variables";
+@import "header";
+@import "stars";
+@import "card";
+@import "catalog";
+
+* {
```

Comment:
Applying a transition to all elements is not recommended because it can cause performance issues. It's better to apply transitions only to specific elements that require them.